salesapp.js

```diff
@@ -11,28 +11,34 @@ function CookieStore(storeName, minHourCustomers, maxHourCustomers, avgCookiesPe
 };
 //-------------------------------FORM----------------------------
 var storeFormEl = document.getElementById('new-store-form');
-var formStore;
-storeFormEl.addEventListener('submit', handleSubmit, false);
-var formStore;
+
+storeFormEl.addEventListener('submit', handleSubmit);
 function handleSubmit(event){
     // var formStore = new CookieStore(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
   event.preventDefault();
   event.stopPropagation(); //stops all bubbling/capturing
   var formNewStoreName = event.target.cookieStoreName.value;
   var formMinCustomers = parseInt(event.target.minCust.value);
   var formMaxCustomers = parseInt(event.target.maxCust.value);
-  var formAvgCookies = parseInt(event.target.avgCust.value);
+  var formAvgCookies = parseFloat(event.target.avgCust.value);
 
    //VERY IMPORTANT! This function pulls the data in the selected field.
   //event = whole element, target = storeFormEl, cookieStoreName = the specific input node, value = input inside of text field.
   // console.log(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
-  formStore = new CookieStore(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
+  var formStore = new CookieStore(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
   console.log(formStore);
   console.log('User pressed submit button on form');
+  console.log(storeArray);
+  formStoreArray.push(formStore);
+  formStore.getAvgCookieCount();
+  formStore.avgSales();
+  formStore.sumSales();
+  formStore.populateTable();
 
-
+};
 
 //-------------------------------METHODS---------------------------------
+
 CookieStore.prototype.getAvgCookieCount = function() {
```

Comment:
Lines where you attach methods to the prototype should probably go right below where you define the constructor function for clarity's sake.